packages/node/src/channel.ts

```diff
@@ -0,0 +1,21 @@
+import * as cf from "@counterfactual/cf.js";
+
+// Not strictly needed, added for easier legibility
```

Comment:
probably bad variable naming if this comment exists

Reply:
This will probably not even be needed. Will remove.